src/docs/sphinx/VariorumTools.rst

```diff
@@ -0,0 +1,62 @@
+.. # Copyright 2019-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
License should start with 2021

Reply:
Bit confused. I was following what you did in your license update...Should this be "2021" only or "2021-2019"? That'd seem odd.

Reply:
The start year should be the year the file was added to the repo. So this file next year will be updated to `2021-2022`, and other files will be `2019-2022`, etc.

Reply:
Got it, so I made the right change then. Take a look at the new commits.

Reply:
Yep the header looks correct!